velox/vector/arrow/Bridge.cpp

```diff
@@ -511,6 +529,54 @@ VectorPtr createFlatVector(
 using WrapInBufferViewFunc =
     std::function<BufferPtr(const void* buffer, size_t length)>;
 
+VectorPtr createStringFlatVectorFromUtf8View(
+    memory::MemoryPool* pool,
+    const TypePtr& type,
+    BufferPtr nulls,
+    const ArrowArray& arrowArray,
+    WrapInBufferViewFunc wrapInBufferView) {
+  int num_buffers = arrowArray.n_buffers;
+
+  // The last C data buffer stores buffer sizes
+  auto* bufferSizes = (uint64_t*)arrowArray.buffers[num_buffers - 1];
+  std::vector<BufferPtr> stringViewBuffers(num_buffers - 3);
+
+  for (int32_t buffer_id = 2; buffer_id < num_buffers - 1; ++buffer_id) {
```

Comment:
could use a comment here explaining what the two first buffers are (and why we skip them), for reader clarity

Reply:
Done